contrib/devtools/utxo_snapshot.sh

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env bash
 #
-# Copyright (c) 2019 The Bitcoin Core developers
+# Copyright (c) 2019-2023 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 #
 export LC_ALL=C
 
 set -ueo pipefail
 
+NETWORK_DISABLED=false
```

Comment:
In commit "script: Enhance validations in utxo_snapshot.sh" (11b7269d83a56f919f9dddb7f7c72a96d428627f)

Assignment seems a little out of place here, would suggest moving it below `PRUNE=...` with other variable assignments.